Utils/Dataflow/run/data4es-start

```diff
@@ -4,10 +4,39 @@ SCRIPT_NAME=data4es
 
 BATCH_SIZE=100
 DEBUG=
+IGNORE=
 
 base_dir=$( cd "$(dirname "$(readlink -f "$0")")"; pwd)
 lib="$base_dir/../shell_lib"
 
+usage() {
+  echo "USAGE
+  $(basename $0) [-hl] [--debug] [-i STAGE...]
+
+  Run 'data4es' process of metadata integration from ATLAS systems (ProdSys,
+  AMI, Rucio, ...) to the DKB Elasticsearch.
+
+OPTIONS
+  -h, --help  show this message and exit
+
+  -l, --list  list known stage IDs and exit
+
+  --debug     do not actualy load integrated metadata to the DKB Elasticsearch
+              (will creates files with data flow snapshots, taken between
```

Comment:
1. "Will create files".
2. Maybe "dataflow", in a single word?

Reply:
Point 1 was fixed, what about 2? Was it missed, or judged fine as-is?

Reply:
@Evildoor,
Sorry, I definitely remember writing a reply to your comment, but seems like it never was posted.

> "Will create files".

Fixed in 69d0fa0.

> Maybe "dataflow", in a single word?

We often use "dataflow" to refer to the integration process -- data processing stages, being executed in a proper order. Here I meant the flow of data through the stages, not the stages themselves -- so the two-words spelling was intentional. If it looks confusing, I can replace it with "flow of data".

Reply:
Aha, I see. Let's leave it as-is then.